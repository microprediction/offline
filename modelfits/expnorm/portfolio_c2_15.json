{"g1": 0.04410626379629048, "g2": 0.429518210891382, "logK": 0.6001056737348687, "loc": -0.04729148558573462, "logScale": 0.1986672484048739}