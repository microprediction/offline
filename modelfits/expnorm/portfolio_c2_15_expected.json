{"g1": 0.34198732780157126, "g2": 3.3395979219215235, "logK": 0.8075237721666576, "loc": -0.0461798734943704, "logScale": -2.016913027676877}